src/Report/Html/Facade.php

```diff
@@ -120,35 +120,22 @@ private function copyFiles(string $target): void
     {
         $dir = $this->getDirectory($target . '.css');
 
-        \file_put_contents(
-            $dir . 'bootstrap.min.css',
-            \str_replace(
-                'url(../fonts/',
-                'url(../.fonts/',
-                \file_get_contents($this->templatePath . 'css/bootstrap.min.css')
-            )
-
-        );
-
+        \copy($this->templatePath . 'css/bootstrap.min.css', $dir . 'bootstrap.min.css');
         \copy($this->templatePath . 'css/nv.d3.min.css', $dir . 'nv.d3.min.css');
         \copy($this->templatePath . 'css/style.css', $dir . 'style.css');
         \copy($this->templatePath . 'css/custom.css', $dir . 'custom.css');
+        \copy($this->templatePath . 'css/octicons.css', $dir . 'octicons.css');
```

Comment:
Glyphicons is no longer part of bootstrap, so I opted for [octicons from Github](https://octicons.github.com/) instead because:

1. It's smaller, just some small SVG files instead of large font files
2. The icons are a bit nicer and most programmers should be familiar with them as GitHub uses them as well